packages/react/src/components/Table/TableBody/SkeletonRow.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { TableCell } from 'carbon-components-react';
+import PropTypes from 'prop-types';
+
+import { SkeletonText } from '../../SkeletonText';
+import { TableColumnsPropTypes } from '../TablePropTypes';
+import { HtmlElementRefProp } from '../../../constants/SharedPropTypes';
+
+const propTypes = {
+  /** The unique row id */
+  id: PropTypes.string.isRequired,
+  /** The unique id for the table */
+  tableId: PropTypes.string.isRequired,
+  /** The unique id for the table test */
+  testId: PropTypes.string.isRequired,
+  columns: TableColumnsPropTypes,
+  /** since some columns might not be currently visible */
+  hasRowSelection: PropTypes.oneOf(['multi', 'single', false]),
+  hasRowExpansion: PropTypes.bool,
+  hasRowNesting: PropTypes.oneOfType([
+    PropTypes.bool,
+    PropTypes.shape({
+      /** If the hierarchy only has 1 nested level of children */
+      hasSingleNestedHierarchy: PropTypes.bool,
+    }),
+  ]),
+  hasRowActions: PropTypes.bool,
+  /** shows an additional column that can expand/shrink as the table is resized  */
+  showExpanderColumn: PropTypes.bool,
+  rowVisibilityRef: HtmlElementRefProp,
+};
+
+const defaultProps = {
+  columns: [],
+  hasRowActions: false,
+  hasRowExpansion: false,
+  hasRowNesting: false,
+  hasRowSelection: false,
+  showExpanderColumn: false,
+  rowVisibilityRef: undefined,
+};
+
+const SkeletonRow = ({
+  id,
+  tableId,
+  testId,
+  rowVisibilityRef,
+  hasRowActions,
+  hasRowExpansion,
+  hasRowNesting,
+  hasRowSelection,
+  showExpanderColumn,
+  columns,
+}) => {
+  return (
+    <tr key={`lazy-row-${id}`} ref={rowVisibilityRef} data-testid={`${tableId}-${testId}-skeleton`}>
```

Comment:
```suggestion
    <tr key={`lazy-row-${id}`} ref={rowVisibilityRef} data-testid={`${tableId}-${testId}`}>
```

Based on my [comment here](https://github.com/carbon-design-system/carbon-addons-iot-react/pull/3239#discussion_r797053340), you can change this, and then set the default `testId` to be `skeleton`.